apps/site/site.json

```diff
@@ -28,9 +28,9 @@
   ],
   "websiteBanners": {
     "index": {
-      "startDate": "2025-07-08T03:00:00.000Z",
-      "endDate": "2025-07-15T03:00:00.000Z",
-      "text": "New security releases to be made available Tuesday, July 15, 2025",
+      "startDate": "2025-07-15T00:00:00.000Z",
+      "endDate": "2025-07-22T00:00:00.000Z",
+      "text": "July Security Release is available",
```

Comment:
```suggestion
      "text": "July Security Releases are available",
```

Reply:
We have always used Security Release **is** available. I think it's one security release that affects multiple lines... Idk

Reply:
Ah okay, in main it is currently using plural (as does the blog post), so was just suggesting this for grammatical consistency